src/config.rs

```diff
@@ -47,16 +47,26 @@ impl Config {
     }
 
     pub fn path() -> Result<PathBuf> {
-        let mut retval = match dirs::config_dir() {
-            Some(path) => path,
-            None => bail!("Could not find the configuration path"),
+        let mut user_path =
+            dirs::config_dir().ok_or_else(|| anyhow!("Could not find the configuration path"))?;
+        let mut system_path = PathBuf::from("etc/xdg");
```

Comment:
Shouldn't it be `/etc/xdg`?

Reply:
Oh! Good catch! Thanks for noticing this. I've fixed it in `main`.